### PR TITLE
Issue 1404 implement test for acl layer log package

### PR DESCRIPTION
**Pull request by youkofan**
Implementing tests for ACL layer (in the log package) -  https://github.com/epam/cloud-pipeline/issues/1404.
Implemented tests of an Acl layer for the log package (for LogApiService class), implemented Spring configuration and also fixed tests in acl.run package in accordance with implementation tests in acl.log package.

### CODE REVIEW

File: api/src/main/java/com/epam/pipeline/manager/region/CloudRegionApiService.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import com.epam.pipeline.entity.info.CloudRegionInfo;
 import com.epam.pipeline.entity.region.AbstractCloudRegion;
 import com.epam.pipeline.entity.region.CloudProvider;
+import com.epam.pipeline.manager.region.CloudRegionManager;
```
Comment (ekazachkova) [inline api/src/main/java/com/epam/pipeline/manager/region/CloudRegionApiService.java]: Could we remove Cloud Region related logic from this PR?

File: api/src/test/java/com/epam/pipeline/acl/log/LogApiServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.when;
+
+public class LogApiServiceTest extends AbstractAclTest {
+
+    @Autowired
+    private LogApiService logApiService;
+
+    @Autowired
+    private LogManager logManager;
```
Comment (ekazachkova) [inline api/src/test/java/com/epam/pipeline/acl/log/LogApiServiceTest.java]: Could we rename this field to `mockLogManager`?

**Review by ekazachkova — CHANGES_REQUESTED** [review]
1. Could you please add link to issue to the PR description?
2. Could you please run all tests and fix failed tests (`RunApiServiceTest` e.g.) ?

**Comment by youkofan** [comment]
> 1. Could you please add link to issue to the PR description?
> 2. Could you please run all tests and fix failed tests (`RunApiServiceTest` e.g.) ?

1. Link added to description.
2. Tests in acl.run package fixed (i also started tests in other packages - they all worked).

File: api/src/main/java/com/epam/pipeline/manager/region/CloudRegionApiService.java (hunk 1 above)
Comment (youkofan) [reply api/src/main/java/com/epam/pipeline/manager/region/CloudRegionApiService.java]: Done. This changes will be shown in another branch.

File: api/src/test/java/com/epam/pipeline/acl/log/LogApiServiceTest.java (hunk 2 above)
Comment (youkofan) [reply api/src/test/java/com/epam/pipeline/acl/log/LogApiServiceTest.java]: I did it, but i think that field with annotation Autowired - it's not literally mock, or am i wrong?

Comment (ekazachkova) [reply api/src/test/java/com/epam/pipeline/acl/log/LogApiServiceTest.java]: This bean was mocked into `AclTestBeans` class (see [line](https://github.com/epam/cloud-pipeline/pull/1442/files#diff-8936017522dedc78b4e2af182bc056ceR114))

**Review by ekazachkova — APPROVED** [review]

**Review by mzueva — APPROVED** [review]

**Pull request merged**